_posts/2015-04-30-foreman-community-newsletter-april-2015.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: post
+title: Foreman Community Newsletter - April 2015
+date: '2015-04-30T16:54:00.000+03:00'
+author: Ori Rabin
+tags:
+- newsletter
+modified_time: '2015-04-30T16:56:07.252+03:00'
+blogger_id: tag:blogger.com,1999:blog-6789674575954398874.post-909076368941673869
+blogger_orig_url: http://blog.theforeman.org/2015/04/foreman-community-newsletter-april-2015.html
+---
+
+<span style="background-color: white; color: black; font-family: Arial; font-size: 20px; font-style: normal; font-variant: normal; font-weight: normal; text-decoration: none; vertical-align: baseline; white-space: pre-wrap;">Foreman Community Newsletter - </span><span style="background-color: transparent; color: black; font-family: 'Trebuchet MS'; font-size: 21px; font-style: normal; font-variant: normal; font-weight: normal; text-decoration: none; vertical-align: baseline; white-space: pre-wrap;">April 2015</span>
```

Comment:
While I don't want to completely rewrite the blog posts, this one's a formatting mess...